src/app-layer-protos.h

```diff
@@ -69,20 +74,18 @@ enum AppProtoEnum {
     // HTTP for any version (ALPROTO_HTTP1 (version 1) or ALPROTO_HTTP2)
     ALPROTO_HTTP,
 
-    /* used by the probing parser when alproto detection fails
-     * permanently for that particular stream */
-    ALPROTO_FAILED,
     /* keep last */
-    ALPROTO_MAX,
+    ALPROTO_MAX_STATIC,
```

Comment:
Can you add a comment that after this there will be dynamic id's?

Reply:
ok